plugin/Record/RecordManagerReadListViewAddress.php

```diff
@@ -112,9 +119,13 @@ public function getRowByName($name)
 		$pWpDb = $this->getWpdb();
 		$mainTable = $this->getMainTable();
 
-		$sql = "SELECT *
-				FROM {$prefix}{$mainTable}
-				WHERE `name` = '".esc_sql($name)."'";
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM %i
+			WHERE `name` = %s",
+			$prefix.$mainTable,
+			esc_sql($name)
```

Comment:
Copy/Paste-Fehler, esc_sql brauchst du nicht verwenden, prepare() übernimmt das Escaping schon 
```suggestion
			$name
```